### PR TITLE
Fixes nightly problems for rpm packages because of new columnar files

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: citus.spec [hunk 1]
```diff
@@ -60,6 +60,19 @@ echo %{pginstdir}/lib/%{sname}.so >> installation_files.list
...
+if [ ${#columnar_sql_files[@]} -gt 0 ]; then
+    echo %{pginstdir}/share/extension/columnar-*.sql >> installation_files.list
+fi
+
+citus_columnar_sql_files=(`find %{buildroot}%{pginstdir}/share/extension -maxdepth 1 -name "citus_columnar-*.sql"`)
+if [ ${#citus_columnar_sql_files[@]} -gt 0 ]; then
+    echo %{pginstdir}/share/extension/citus_columnar-*.sql >> installation_files.list
+fi
+
+[[ -f %{pginstdir}/share/extension/citus_columnar.control ]] && echo %{pginstdir}/share/extension/citus_columnar.control >> installation_files.list
```
Comment (onurctirtir) [inline citus.spec]: why do we have two lines for citus_columnar.control L64 & L75

**Review by onurctirtir — COMMENTED** [review]
And definitely need to locally-build & test el7 package, e.g.: whether we can install it and then can do `create extension citus;` via psql.

File: citus.spec (hunk 1 above)
Comment (gurkanindibay) [reply citus.spec]: my fault. Removed it.

**Comment by gurkanindibay** [comment]
> And definitely need to locally-build & test el7 package, e.g.: whether we can install it and then can do `create extension citus;` via psql.

Tested successfully

**Review by onurctirtir — APPROVED** [review]

**Review by hanefi — APPROVED** [review]

**Pull request merged**